perception/autoware_probabilistic_occupancy_grid_map/synchronized_grid_map_fusion.md

```diff
@@ -4,12 +4,6 @@
 
 This package is used to fuse the OGMs from synchronized sensors. Especially for the lidar.
 
-Here shows the example OGM for the this synchronized OGM fusion.
-
-| left lidar OGM                    | right lidar OGM                     | top lidar OGM                   |
-| --------------------------------- | ----------------------------------- | ------------------------------- |
-| ![left](image/left_lidar_ogm.png) | ![right](image/right_lidar_ogm.png) | ![top](image/top_lidar_ogm.png) |
-
```

Comment:
- @YoshiRi -san https://github.com/autowarefoundation/autoware_universe/pull/5993/files#r2647091131 please check here, these images were never included in the repository.